testData/solutions/linux.json

```diff
@@ -1430,15 +1430,61 @@
             "configuration": {
                 "type": "gpii.xrandr",
                 "capabilities": [
-                    "display.screenEnhacements",
+                    "display.screenEnhancement",
                     "applications.org\\.freedesktop\\.xrandr.id"
-                ]
-           }
+                ],
+                "capabilitiesTransformations": {
+                    "screen-resolution": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/screenResolution",
+                            "defaultInputValue": "normal",
+                            "options": {
+                                "high": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
+                                        }
+                                    }
+                                },
+                                "normal": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
```

Comment:
We should fix this in the settings handler, so that it doesn't try to set a resolution larger than the screen supports, rather than screw this up in the transformations for everyone